src/index.html

```diff
@@ -10,13 +10,51 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header"
+      id="top"
+    >
+      <a>
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <a class="nav__link is-active">Apple</a>
+        <a class="nav__link">Samsung</a>
+        <a class="nav__link">Smartphones</a>
+        <a
+          class="nav__link"
+          data-qa="hover"
```

Comment:
The `data-qa` attribute is used for testing purposes, such as identifying elements in automated tests. Ensure that this attribute is necessary for your project. If not, consider removing it to keep the HTML clean.